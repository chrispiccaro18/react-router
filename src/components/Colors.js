import React from 'react';
import PropTypes from 'prop-types';

function Colors({ match }) {
  const color = match.params.color;
  const divStyle = {
    width: '500px',
    height: '500px',
    backgroundColor: color
  };
  return <div style={divStyle}></div>;
}

Colors.propTypes = {
  match: PropTypes.object.isRequired
};

export default Colors;
