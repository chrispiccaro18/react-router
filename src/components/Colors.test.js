import React from 'react';
import { shallow } from 'enzyme';
import Colors from './Colors';

describe('Colors component', () => {
  it('renders Colors', () => {
    const match = {
      params: {
        color: 'blue'
      }
    };
    const wrapper = shallow(<Colors match={match} />);
    expect(wrapper).toMatchSnapshot();
  }); 
});
