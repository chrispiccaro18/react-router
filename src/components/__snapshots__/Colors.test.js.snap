// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Colors component renders Colors 1`] = `
<div
  style={
    Object {
      "backgroundColor": "blue",
      "height": "500px",
      "width": "500px",
    }
  }
/>
`;
